<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpacerConfiguration</class>
 <widget class="QDialog" name="SpacerConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spacer Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelSize">
     <property name="text">
      <string>Space width:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="sizeSB">
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>2048</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Space type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="sizeFixedRB">
     <property name="text">
      <string>fixed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="typeCB">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QRadioButton" name="sizeExpandRB">
     <property name="text">
      <string>expandable</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>SpacerConfiguration</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeFixedRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
